homeassistant/components/go2rtc/__init__.py

```diff
@@ -72,9 +72,15 @@
     )
 )
 
-
 CONFIG_SCHEMA = vol.Schema(
-    {DOMAIN: vol.Schema({vol.Optional(CONF_URL): cv.url})},
+    {
+        DOMAIN: vol.Schema(
+            {
+                vol.Exclusive(CONF_URL, DOMAIN, EXCLUSE_ERROR_MESSAGE): cv.url,
```

Comment:
What's `EXCLUSE`? Do you mean `EXCLUDE`?

Reply:
Typo it should be called exclusive